specerror/config-linux.go

```diff
@@ -10,14 +10,21 @@ import (
 const (
 	// DefaultFilesystems represents "The following filesystems SHOULD be made available in each container's filesystem:"
 	DefaultFilesystems = "The following filesystems SHOULD be made available in each container's filesystem:"
+	NamespacesType     = "The following namespace types are supported:"
+	NamespacesPath     = "This value MUST be an absolute path."
```

Comment:
I prefer to extend it to "This value MUST be an absolute path in the runtime mount namespace. "
There are several 'an absolute path' RFCs,  https://github.com/opencontainers/runtime-tools/blob/master/specerror/config.go#L32,  https://github.com/opencontainers/runtime-tools/blob/master/specerror/config.go#L42.
It will be easier to differentiate them.